{
    "error": "HTTPSConnectionPool(host='alioth.debian.org', port=443): Max retries exceeded with url: /plugins/scmgit/cgi-bin/gitweb.cgi?p=apt/apt.git;a=blob;f=debian/changelog;hb=HEAD (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7fdb92002940>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
}